docs/spec/distribution/overview.md

```diff
@@ -2,71 +2,72 @@
    change).
- - Whenever a validator chooses to change the commission on rewards, all accumulated 
+- Whenever a validator chooses to change the commission on rewards, all accumulated
    commission rewards must be simultaneously withdrawn.
 
 The above scenarios are covered in `hooks.md`.
 
-The distribution mechanism outlines herein is used to lazily distribute the
+The distribution mechanism outlines herein are used to lazily distribute the
 following rewards between validators and associated delegators:
- - multi-token fees to be socially distributed, 
- - proposer reward pool, 
- - inflated atom provisions, and
- - validator commission on all rewards earned by their delegators stake
+
+- multi-token fees to be socially distributed
+- proposer reward pool
+- inflated atom provisions
+- validator commission on all rewards earned by their delegators stake
 
 Fees are pooled within a global pool, as well as validator specific
 proposer-reward pools. The mechanisms used allow for validators and delegators
 to independently and lazily withdraw their rewards.  
 
-## Shortcomings 
+## Shortcomings
 
 As a part of the lazy computations, each delegator holds an accumulation term
 specific to each validator which is used to estimate what their approximate
-fair portion of tokens held in the global fee pool is owed to them. 
+fair portion of tokens held in the global fee pool is owed to them.
 
 ```
 entitlement = delegator-accumulation / all-delegators-accumulation
 ```
 
-Under the circumstance that there were constant and equal flow of incoming
+Under the circumstance that there were constant and equal flows of incoming
 reward tokens every block, this distribution mechanism would be equal to the
 active distribution (distribute individually to all delegators each block).
-However this is unrealistic so deviations from the active distribution will
+However, this is unrealistic so deviations from the active distribution will
 occur based on fluctuations of incoming reward tokens as well as timing of
-reward withdrawal by other delegators. 
+reward withdrawal by other delegators.
 
-If you happen to know that incoming rewards are about significantly move up,
+If you happen to know that incoming rewards are about significantly go up,
 you are incentivized to not withdraw until after this event, increasing the
 worth of your existing _accum_.
 
 ## Affect on Staking
 
 Charging commission on Atom provisions while also allowing for Atom-provisions
 to be auto-bonded (distributed directly to the validators bonded stake) is
-problematic within DPoS. Fundamentally these two mechanisms are mutually
+problematic within DPoS. Fundamentally, these two mechanisms are mutually
 exclusive. If there are Atom commissions and auto-bonding Atoms, the portion
-of Atoms the reward distribution calculation would become very large as the Atom
+of Atoms in the reward distribution calculation would become very large as the Atom
```

Comment:
Also, I find the following to be a mouthful and somewhat hard to grasp:

```
If there are Atom commissions and auto-bonding Atoms, the portion
of Atoms in the reward distribution calculation would become very large as the Atom
portion for each delegator would change each block making a withdrawal of rewards
for a delegator require a calculation for every single block since the last
withdrawal.
```

Maybe it's just me? 😅

Reply:
No, it could be better phrased, maybe: `If there are both commissions and auto-bonding on the staking token, the amount of staking tokens each validator and delegator has would change each block, so we would have to iterate through all of them`

cc @rigelrozanski